crates/hyfetch/Cargo.toml

```diff
@@ -15,6 +15,7 @@ ansi_colours = { workspace = true, features = [] }
 anstream = { workspace = true, features = [], optional = true }
 anyhow = { workspace = true, features = ["std"] }
 bpaf = { workspace = true, features = [] }
+clearscreen = "3.0.0"
```

Comment:
This should be removed. Just use our own `color_util::clear_screen` function. (It's *really* simple btw...)